components-compose/src/main/java/uk/gov/hmrc/components/compose/organism/headline/HeadlineCardView.kt

```diff
@@ -0,0 +1,157 @@
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.ripple.LocalRippleTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import uk.gov.hmrc.components.compose.R
+import uk.gov.hmrc.components.compose.atom.heading.Heading3
+import uk.gov.hmrc.components.compose.atom.heading.Heading5
+import uk.gov.hmrc.components.compose.ui.theme.HmrcRippleTheme
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
+
+object HeadlineCardView {
+    @Composable
+    operator fun invoke(
+        modifier: Modifier = Modifier,
+        headline: String,
+        title: String,
+        childPadding: Boolean = true,
+        views: List<@Composable () -> Unit> = listOf(),
+        chevronContentDescription: String = stringResource(id = R.string.accessibility_button_activate),
+        onHeadlineCardClick: (() -> Unit)? = null
+    ) {
+        if (onHeadlineCardClick != null) {
+            CompositionLocalProvider(LocalRippleTheme provides HmrcRippleTheme) {
+                Row(
+                    modifier = Modifier
+                        .background(HmrcTheme.colors.hmrcWhiteBackground)
+                        .clickable {
+                            onHeadlineCardClick()
+                        }
+                ) {
+                    HeadlineCard(
+                        modifier = modifier.then(
+                            Modifier
+                                .weight(1f)
+                                .semantics(mergeDescendants = true) {}
+                        ),
+                        headline,
+                        title,
+                        childPadding,
+                        views
+                    )
+                    Image(
+                        modifier = Modifier
+                            .align(Alignment.CenterVertically)
+                            .semantics(mergeDescendants = true) {},
+                        painter = painterResource(id = R.drawable.components_ic_chevron_right),
+                        contentDescription = chevronContentDescription
+                    )
+                    Spacer(modifier = Modifier.width(HmrcTheme.dimensions.hmrcSpacing8))
+                }
+            }
+        } else {
+            HeadlineCard(modifier, headline, title, childPadding, views)
+        }
+    }
+
+    @Composable
+    fun HeadlineCard(
+        modifier: Modifier = Modifier,
+        headline: String,
+        title: String,
+        childPadding: Boolean = true,
+        views: List<@Composable () -> Unit> = listOf()
```

Comment:
I think we can restrict this to a ColumnScope as per my above suggestion. I think we previously added to a LinearLayout so we only need to support the linear vertical option! If we name it `content` then it's a bit more consistent with what we've already built too :) 

```suggestion
        content: @Composable ColumnScope.() -> Unit = {},
```